dvc/command/status.py

```diff
@@ -39,7 +39,7 @@ def _show(self, status, indent=0):
 
         for key, value in status.items():
             if isinstance(value, str):
-                ui.write("{}{}{}".format(ind, self._normalize(value), key))
+                ui.write(f"{ind}{self._normalize(value)}{key}")
```

Comment:
I'm honestly not sure about all of these changes, as I personally find `.format()` more readable in these cases, rather than having function calls inside f-strings.

Problably something the whole @iterative/dvc team needs to weigh in on

Reply:
I too resisted the change to this new format but now, after a year of use, I find f-strings easier to read and write.  It is a choice for your team to make but these lines are shorter on the page and a [3.62 times runtime speedup](https://www.scivision.dev/python-f-string-speed) does not hurt.

Reply:
I guess in this particular case even `format` one was a bit ugly and needed smth like `nvalue = self._normalize(value)` so that then you could use `nvalue` in it instead of putting that chunk of code directly into it. If we adjust it like that new f-string will actually look better than the old format :slightly_smiling_face: